linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_etc_shadow.rule

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
```

Comment:
@iokomin I'm not sure why you duplicated this file. I'd prefer if you instead rename `groupowner_shadow_file.rule` -> `file_groupowner_etc_shadow.rule` and replaced instances of `groupowner_shadow_file` with `file_groupowner_etc_shadow` and updated the corresponding oval checks to add `multi_platform_{rhel,fedora,ol,debian,ubuntu}` and drop the `prodtype` tag from the original.

Reply:
Also make sure to update the corresponding fixes as well. :)

Reply:
Renaming is definitely preferable way. I duplicated these rules this time as I don't have other platforms on hand to verify these changes. This change also require touching profiles in other products. I decided to consult with you guys first, is it OK for me do changes like this or better SSG gurus will bring up everything in sync :)

Reply:
It's fine if you feel comfortable doing this, or else I can create a quick PR to do this and you can rebase on top of it when it merges. :)

If you have questions, we do have an IRC channel ([#openscap](https://www.open-scap.org/resources/support/#irc-freenode)) where a bunch of us lurk.

Reply:
@iokomin Sure! I'm going to propose a WIP `debian8` one first to start getting feedback from @mpreisler and @redhatrises, but then I'll do that.

Reply:
@cipherboy I've moved this PR to WIP. Will be waiting for updates from you to rebase it and fix things to be updated.

Reply:
@iokomin Thanks! I believe #3130 should do what we need :) Let me know if you have any thoughts on it.